.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
You should not include any editor settings in the repo. Please remove `.vscode/settings.json`

Reply:
fixed